deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxTouchFeature.java

```diff
@@ -67,7 +68,7 @@ public void preflight(final Path workdir, final String filename) throws Backgrou
             if(log.isWarnEnabled()) {
                 log.warn(String.format("Target already exists %s/%s", workdir, filename));
             }
-            throw new AccessDeniedException(MessageFormat.format(LocaleFactory.localizedString("Cannot create {0}", "Error"), filename)).withFile(workdir);
+            throw new ConflictException(MessageFormat.format(LocaleFactory.localizedString("Cannot create {0}", "Error"), filename)).withFile(workdir);
```

Comment:
No conflict resolution preflight checks. This is handled by overwrite prompts or sync strategies.